safe/storage/test/test_vector.py

```diff
@@ -1,99 +0,0 @@
-# coding=utf-8
-"""**Tests for safe vector layer class**"""
-
-__author__ = 'Tim Sutton <tim@kartoza.com>'
-__revision__ = '$Format:%H$'
-__date__ = '21/08/2012'
-__license__ = "GPL"
-__copyright__ = 'Copyright 2012, Australia Indonesia Facility for '
-__copyright__ += 'Disaster Reduction'
-
-import os
-import logging
-import unittest
-
-from safe.common.utilities import temp_dir, unique_filename
-from safe.storage.utilities import read_keywords
-from safe.storage.vector import Vector, QGIS_IS_AVAILABLE
-from safe.test.utilities import test_data_path, get_qgis_app
-
-if QGIS_IS_AVAILABLE:   # Import QgsVectorLayer if qgis is available
-    QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
-    from qgis.core import QgsVectorLayer
-
-
-LOGGER = logging.getLogger('InaSAFE')
-KEYWORD_PATH = test_data_path('exposure', 'exposure.xml')
-SQLITE_PATH = test_data_path('exposure', 'exposure.sqlite')
-SHP_BASE = test_data_path('exposure', 'buildings_osm_4326')
-EXPOSURE_SUBLAYER_NAME = 'buildings_osm_4326'
-
-
-class VectorTest(unittest.TestCase):
```

Comment:
Ouch, I think I delete it because all of the unit test need read keywords.
I will create some unit tests that independent from `.keywords` reading

Thanks